components/net/http_loader.rs

```diff
@@ -529,6 +530,25 @@ pub fn modify_request_headers(headers: &mut Headers,
     // https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch step 11
     if load_data.credentials_flag {
```

Comment:
Where is the `credentials_flag` set? AFAICT it shouldn't have ever been set, because `send-authentication-basic.htm` never called `withCredentials` and hence the request should not have the `Authorization` header at all, yet you were receiving a `responseText` of `(token)\npass`.